terraform/variables.tf

```diff
@@ -151,6 +151,11 @@ variable "service_updates_severities" {
   default = ["critical", "important"]
 }
 
+variable "service_updates_types" {
+  type    = list(string)
+  default = ["engine-update", "security-update"]
```

Comment:
do we need to duplicate these default values in multiple places?

Reply:
This variable is auto-generated, and the `default` value is not in use.